config/features.yml

```diff
@@ -150,6 +150,10 @@ features:
     actor_type: user
     description: Correctly formats the `va:attachments` XML for file uploads
     enable_in_development: true
+  ezr_form_prefill_with_providers_and_dependents:
+    actor_type: user
+    description: Adds insurance providers and dependents to ezr prefill data
+    enable_in_development: true
```

Comment:
I added a flipper because the json schema needs to be fixed in order for the frontend to work correctly.